### ISSUE TITLE
GPU Compile error on PPO with MaskedPPOTrajectory

### ISSUE THREAD

**Issue by qwjyh** [JuliaReinforcementLearning/ReinforcementLearning.jl#1007]
When I ran PPO with MaskedPPOTrajectory on CUDA, it failed with a GPU compile error.
The code doesn't send a mask to GPU memory so adding a line to l.160 of `src/ReinforcementLearningZoo/src/algorithms/policy_gradient/ppo.jl` solved the problem.
```diff
function RLBase.prob(p::PPOPolicy{<:ActorCritic,Categorical}, state::AbstractArray, mask)
    logits = p.approximator.actor(send_to_device(device(p.approximator), state))
+ mask = send_to_device(device(p.approximator), mask)
    if !isnothing(mask)
        logits .+= ifelse.(mask, 0.0f0, typemin(Float32))
    end
```
